pose-detection/src/calculators/keypoints_one_euro_filter.ts

```diff
@@ -51,7 +51,7 @@ export class KeypointsOneEuroFilter implements KeypointsFilter {
     // If value is too small smoothing will be disabled and keypoints will be
     // returned as is.
     let valueScale = 1;
-    if (this.config.minAllowedObjectScale != null) {
+    if (!this.config.disableValueScaling) {
```

Comment:
That should be in previous CL

Reply:
This change was temporary and is now added and used in Add 3D keypoints output to TFJS blazepose commit